.github/workflows/feature-serverless-framework-deployment.yml

```diff
@@ -198,10 +270,10 @@ jobs:
         with:
           go-version: 1.21
 
-      - name: Set up Node 18
+      - name: Set up Node.js 16.16.0
```

Comment:
Any particular reason Node had to be downgraded? Is Node 18 not compatible with the `serverless-aliyun-function-compute` plugin?

Reply:
Nope, I believe they are compatible as well. It's just that I saw you upgrading the Node version to 16.6.0 for all the other integration and end-to-end tests (e.g. AWS, GCR, Cloudflare) so I thought it would be good to keep it consistent with that. Would you prefer that I leave it at version 18?

Reply:
Ok got it, we can just leave it at 16.6.0 for now